### PR TITLE
Houndoomite and Manectite

### PR DESCRIPTION
**Pull request by Leafsw0rd**
- Introduces two new Temporary Battles that spawn on Kalos overworld and award mega stones - the "Wild Electrike Horde" and the "Wild Houndour Horde".
- Unlocking the fights requires the player to beat the Champion, capture 500 of the pre-evolution and then be in the appropriate weather that would normally buff the Pokemon (Sunny for Houndoom, Thunderstorm for Manectric)
- Added NPCs to Dendemille town to hint at the acquisition method - the Worrisome Wolf-Loving Woman and the Dubious Dog-loving Duke.
- Added NPC Dexio to Lumiose City, who tells the player where to find the more detailed statistics in the Pokedex, as Gengarite, Houndoomite and Manectite unlock based on statistics visible there. Dexio uses a new sprite based on his Gen 6 appearance, which has been credited.

### CODE REVIEW

File: src/scripts/towns/TownList.ts [hunk 1]
```diff
@@ -4153,6 +4162,24 @@ const LaverreGengariteAster2 = new NPC('Hex Maniac Aster', [
...
+    'You could build your own Houndour pack and find out, but if it\'s strong enough to call five hundred Pokémon, it probably wouldn\'t give anyone weaker than Diantha the time of day.',
+], {image: 'assets/images/trainers/Gentleman.png',
+});
+
+const DendemilleWolfLover = new NPC('Worrisome Wolf-loving Woman', [
+    'I adore wolf Pokémon! The most precious of all have to be the brave little Electrike - I\'ve gathered five hundred of the darlings in my home.',
+    'But now that I have done so, their behaviour has taken a... regrettable turn.',
+    'When storms come along the horizon, they\'ll run en masse out to Route 16.',
+    'They fortunately do return home when the weather changes, at least. I wonder what sort of creature is calling them? A strong Pokémon, perhaps?',
+    'I\'ve been debating asking someone else to collect five hundred of their own Electrike and investigate, but if it can call such a fearsome pack with it\'s presence alone, I don\'t imagine anyone less than Diantha herself could handle it.',
```
Comment (zyx-xyzzy) [inline src/scripts/towns/TownList.ts:4179]: it’s presence -> its presence

Comment (Leafsw0rd) [reply src/scripts/towns/TownList.ts]: fixed!

**Comment by Aegyo** [comment]
Conflict in Update.ts

**Comment by Leafsw0rd** [comment]
> Conflict in Update.ts

should be fixed with this update

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Pull request merged**